DashAI/back/metrics/classification/accuracy.py

```diff
@@ -13,12 +13,12 @@ class Accuracy(ClassificationMetric):
     """
 
     @staticmethod
-    def score(true_labels: list, probs_pred_labels: list):
+    def score(true_labels: DashAIDataset, probs_pred_labels: list):
```

Comment:
Check `prepare_to_metric` comments and apply here too.

```suggestion
    def score(dataset: DashAIDataset, probs_pred_labels: np.ndarray) -> float:
```